css-speech-1/Overview.bs

```diff
@@ -568,7 +568,7 @@ The 'pause-before' and 'pause-after' properties</h3>
 
 	<pre class=propdef>
 	Name: pause-before, pause-after
-	Value: &lt;time&gt; | none | x-weak | weak | medium | strong | x-strong
+	Value: &lt;time [0s,∞]&gt; | none | x-weak | weak | medium | strong | x-strong
```

Comment:
`0s` is used instead of `0` for consistency with existing time range notations, but I think it is not required.